src/Components/Socials.js

```diff
@@ -10,19 +10,31 @@ function Socials() {
   const [showProgress, setShowProgress] = useState(true)
   const { username } = useParams()
   const [profile, setProfile] = useState({})
+  const [profileFound, setProfileFound] = useState(false)
 
   useEffect(() => {
     fetch(`/data/${username}.json`)
       .then((response) => response.json())
-      .then((data) => setProfile(data))
+      .then((data) => {
+        setProfile(data)
+        setProfileFound(true)
+      })
       .catch((error) => {
         console.log('Socials useEffect', error)
-        alert('An error occurred please try again later.')
+        setProfileFound(false)
       })
       .finally(() => setShowProgress(false))
   }, [username])
 
-  return (
+  if (!profileFound) {
+    return <div style={{ textAlign: 'center' }}>
+      <img src="https://d1aettbyeyfilo.cloudfront.net/eddiejaoude/19083469_1619470266Fpchubber-hi.webp" alt="" style={{ width: '150px' }}/>
```

Comment:
Could you perhaps have the photo downloaded, so it is always there?

Reply:
I shall

Reply:
hey @CodingSpecies, can you suggest in which directory do I place the image?

Reply:
@harshmathurx I would perhaps say make an image folder somewhere, or add the image in the public folder, where the icons are. But this is what I suggest, I am not sure what others would say.

Reply:
> @harshmathurx I would perhaps say make an image folder somewhere, or add the image in the public folder, where the icons are. But this is what I suggest, I am not sure what others would say. 

aight, thanks
@rtynski do you have any suggestions about where to place the image?